File: UserWorkflow.Application/Models/Exercise/ExerciseInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UserWorkflow.Application.Models.Exercise
{
    public class ExerciseInfo
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public List<ExerciseRelationModel> SportRelations { get; set; }
        public List<ExerciseRelationModel> TraumaRelations { get; set; }
        public List<ExerciseRelationModel> BodypartRelation { get; set; }
        public List<int> ExerciseTutorialLinks { get; set; }
    }

    public class ExerciseRelationModel 
    {
        public int Id { get; set; }
        public string Name { get; set; }
    }
}

File: esport.ui2/src/entities/exercise/model/types/exercise-info.ts
import { ExerciseRelationModel } from "./exercise-relation-model";

export interface IExerciseInfo {
  name: string;
  description: string;
  sportRelations: ExerciseRelationModel[];
  traumaRelations: ExerciseRelationModel[];
  bodypartRelation: ExerciseRelationModel[];
  exerciseTutorialLinks: number[];
}
